src/spacy/data.clj

```diff
@@ -2,4 +2,5 @@
 
 (defprotocol Events
   (fetch [this slug])
+  (all-slugs [this])
```

Comment:
I think this is good, but it kind of bothers me that `all-slugs` returns both a name and a slug. But I can't really think of a better name for it, since we aren't fetching the whole event, so I'm ok with it. 🤔